module/os/freebsd/spl/spl_uio.c

```diff
@@ -45,6 +45,16 @@
 #include <sys/vnode.h>
 #include <sys/zfs_znode.h>
 
+static void
+zfs_freeuio(struct uio *uio)
+{
+#if __FreeBSD_version > 1500013
```

Comment:
just double checking, should this be `>=` or `>`?

Reply:
`>` is correct. `__FreeBSD_version` was not bumped when freeuio() was added to FreeBSD, and its value at the time was 1500013. This change won't compile for previous revisions after the bump from 1500012 to 1500013, to be safe we can just consider 1500014 to be the first version which includes freeuio().